### ISSUE TITLE
Additional Studio Stats

### ISSUE THREAD

**Issue by AeroKoder** [ScratchAddons/ScratchAddons#7974]
### Why this would be helpful

This addon would be helpful when wanting to know the creation date, and other additional stats, of a studio.

### How this addon works

This addons displays additional stats on the project and studio pages.

### Additional context

Studio Screenshot:
![Screenshot 2024-12-01 170132](https://github.com/user-attachments/assets/03aed6e1-8b3b-4b70-94ee-b3b14af05683)

**Comment by Samq64** [comment]
In the future could you open an issue ahead of time, so the idea can be discussed before you submit a pull request?

**Comment by AeroKoder** [comment]
> In the future could you open an issue ahead of time, so the idea can be discussed before you submit a pull request?

Yes, I just got a GitHub account and already had the code created.
